### DISCUSSION TITLE
Deployment

### DISCUSSION THREAD

**Discussion by gobinda05** [community/community#57728] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Why all of my deployment are not secured from last week?

All website is saying :- 
Deceptive site ahead
Attackers on deygobinda.github.io may trick you into doing something dangerous like installing software or revealing your personal information (for example, passwords, phone numbers, or credit cards). [Learn more](chrome-error://chromewebdata/#)
Google Safe Browsing recently [detected phishing](chrome-error://chromewebdata/#) on deygobinda.github.io. Phishing sites pretend to be other websites to trick you.

You can [report a detection problem](chrome-error://chromewebdata/#) or, if you understand the risks to your security, [visit this unsafe site](chrome-error://chromewebdata/#).

how to solve it

**Comment by airtower-luna** [comment, marked as answer]
I notice you have a https://github.com/deygobinda/Netflix-LandingPage repository, that or some other imitation you made probably got reported for phishing and ended up on the Google Safe Browsing blocklist. I saw a few people ask about this in the last days, and they all had a Netflix clone. Is there some irresponsible teacher/course that tells people to make those? :thinking: 

See the Google Safe Browsing documentation on [Social engineering (phishing and deceptive sites)](https://developers.google.com/search/docs/advanced/security/social-engineering?hl=en) for what you can do, in very short it's to remove the site and *then* [request a review](https://safebrowsing.google.com/safebrowsing/report_error/?hl=de).

GitHub support *can't* help there, because the warning isn't created by your Pages site. It's your *browser* telling you "this site is on the Google Safe Browsing list of phishing sites".

**Reply by gobinda05** [reply]
Thanks
